includes/Checker/Checks/Plugin_Repo/File_Type_Check.php

```diff
@@ -90,6 +90,9 @@ protected function check_files( Check_Result $result, array $files ) {
 		if ( $this->flags & self::TYPE_APPLICATION ) {
 			$this->look_for_application_files( $result, $files );
 		}
+
+		// Check for badly named files.
+		$this->look_for_badly_named_files( $result, $files );
```

Comment:
Can you please wrap this check in a flag check, similar to all other parts of this check above? You could add a new `TYPE_BADLY_NAMED` class constant for that.

Please keep in mind to also update the `TYPE_ALL` constant accordingly.

Reply:
ok thanks Felix. I'll do that.